swagger.yaml

```diff
@@ -0,0 +1,94 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Tech Holding Classes
+  description: An app to manage Tutorials
+host: localhost:8000
+basePath: "/"
+tags:
+- name: tutorials
+  description: API for user in the system
+schemes:
+- http
+consumes:
+- application/yaml
+produces:
+- application/yaml
+paths:
+  tutorial:
+    parameters:
+    - name: id
+      in: path
+      required: true
+      description: ID of tutorial that we want to find
+      type: integer
+    get:
+      tags:
+      - tutorials
+      summary: Get all tutorials in system
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/tutorial'
+    post:
+      tags:
+      - tutorials
+      description: Create new tutorial in system
+      parameters:
+      - name: tutorial
+        in: body
+        description: User that we want to create
+        schema:
+          $ref: '#/definitions/tutorial'
+      produces:
+      - application/yaml
+      responses:
+        '200':
+          description: New user is created
+          schema:
+            $ref: '#/definitions/tutorial'
+    delete:
+      summary: Delete tutorials with given ID
+      tags:
+      - tutorials
+      responses:
+        '200':
+          description: tutorial is deleted
+          schema:
+            $ref: '#/definitions/tutorial'
+    patch:
+      summary: Update tutorial with give ID
+      tags:
+      - tutorials
+      parameters:
+      - name: tutorial
+        in: body
+        description: tutorial with new values of properties
+        schema:
+          $ref: '#/definitions/tutorial'
+      responses:
+        '200':
+          description: tutorial is updated
+          schema:
+            $ref: '#/definitions/tutorial'
+definitions:
+  tutorial:
+    required:
+    - UUID
+    - createdAt
+    - updatedAt
+    properties:
+      UUID:
```

Comment:
The name of this field should be `_id` but the type should be `string` and format should be `uuid`.

```
_id: 
  type: string
  format: uuid
```

Same for id in other places.